### PR TITLE
Updates to be compatible with Ruby 2.6

### PR DESCRIPTION
**Pull request by pnomolos**
- [x] Remove active_record_inline_schema
- [x] Update AR

### CODE REVIEW

**Comment by olivierlacan** [comment]
@pnomolos Is it necessary to make so many stylistic changes unrelated to Ruby 2.6 compatibility in this PR? Could StandardRB stuff be done in a separate PR for clarity?

**Comment by pnomolos** [comment]
> @pnomolos Is it necessary to make so many stylistic changes unrelated to Ruby 2.6 compatibility in this PR? Could StandardRB stuff be done in a separate PR for clarity?

@olivierlacan I suppose it's not and could be done in a different PR.  I guess my thought was that since I'm ripping a lot of stuff out anyway I might as well go whole-hog, but to be fair it would be clearer in a separate PR.  I'll back that off.

> And do you need help getting this to the finish line?

If you have any experience with JRuby that's the only thing blocking this right now.  It looks like it's something likely related to threading (running each failing spec on its own works fine) but I haven't had time to dig in yet.

**Comment by olivierlacan** [comment]
Looking at CI, it seems like native extensions gems are failing on all JRuby builds too. Maybe a new run might solve that?

**Comment by pnomolos** [comment]
> Looking at CI, it seems like native extensions gems are failing on all JRuby builds too. Maybe a new run might solve that?

Yeah, there's something going on with Travis not using the upsert-java.gemspec for JRuby, and locally while I have that working there are two of the speed specs that are failing.

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby-2.2.4
```
Comment (olivierlacan) [inline .ruby-version]: This should be `2.2.4`. No need for `ruby-` in front of it.

Comment (olivierlacan) [inline .ruby-version]:
```suggestion
2.2.4
```

Comment (olivierlacan) [reply .ruby-version]: Also curious why you'd set the Ruby version that far back? 2.2 is EOL. Should be `2.5` or `2.6` at least.

Comment (pnomolos) [reply .ruby-version]: For people who aren't able to upgrade and are still wanting to use Upsert.  That being said, I suppose I'm not really planning on back porting new features anyway.

**Comment by olivierlacan** [comment]
@pnomolos I see green. 🎉 👍

**Comment by pnomolos** [comment]
@olivierlacan I'm going to merge this, then I'm going to merge the PR I've got open to improve the build matrix, then I'll talk to @seamusabshere about getting a gem cut!

**Comment by olivierlacan** [comment]
@pnomolos @seamusabshere Just ran master on our 2.6 test suite and everything's green now. Any idea when we can expect a new release? I'd love to help with the changelog if necessary.

**Comment by pnomolos** [comment]
@olivierlacan I was hoping to get #118 all green and merged before we release.  I'll also need @seamusabshere to cut the gem as I don't have permissions there :)

**Comment by olivierlacan** [comment]
@seamusabshere Any chance you can cut a release for this? I'm not seeing this release on RubyGems yet and 2.6 is bound to be in much wider use now than a few months ago. Since 2.7 is coming up especially this is going to break a lot more as people weary of the latest Ruby minor finally bump up to 2.6.

**Comment by pnomolos** [comment]
@olivierlacan I'll set a reminder to do this tomorrow.  I'll probably cut a 2.99 release that has a few very minor JRuby bugs but at least works on 2.6.

**Pull request merged**